src/session.js

```diff
@@ -47,9 +52,17 @@ function tryReadingIframeLocation(iframe) {
             // we can sort of detect this by checking the location of the iframe
             // if the contentDocument object cannot be accessed, its due to a security error.
             // security error will occur when the frame is on a different origin
-            resolve(iframe.contentDocument.location);
+
+            // Heuristic to detect google auth error in the iframe document
+            var textContent = iframe.contentDocument.body.textContent;
```

Comment:
Changed to `const` (though there are `var`s elsewhere in this file, was an early ES6 adoption)